tr55/model.py

```diff
@@ -77,6 +77,8 @@ def runoff_nrcs(precip, evaptrans, soil_type, land_use):
     The runoff equation from the TR-55 document.  The output is a
     runoff value in inches.
     """
+    if land_use == 'cluster_housing':
```

Comment:
Cluster housing is supposed to be treated like low intensity residential for hydrological purposes.